src/Service/DestroyWithParamsTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ChartMogul\Service;
+
+use ChartMogul\Http\ClientInterface;
+
+/**
+* @codeCoverageIgnore
+*/
+trait DestroyWithParamsTrait
+{
+    /**
+     * Delete a resource
+     * @return boolean
+     */
+    public function destroy(array $params = [], ClientInterface $client = null)
```

Comment:
Like in the `UpdateWithParamsTrait` I would suggest changing the name
```suggestion
    public function destroyWithParams(array $params = [], ClientInterface $client = null)
```